content/rc/security/database-security/network-security.md

```diff
@@ -68,58 +68,15 @@ To restrict a database to a specific set of source IP addresses or subnets:
 
 A [Virtual Private Cloud](https://en.wikipedia.org/wiki/Virtual_private_cloud) (VPC) is an isolated set of resources within a [public cloud](https://en.wikipedia.org/wiki/Cloud_computing#Public_cloud), usually having its own subnets and VLAN.
 
-Databases in Flexible and Annual subscriptions are almost always deployed in a Redis VPC. In most cases, you'll need to create a **VPC peering connection** to access these databases. A VPC peering connection allows unrestricted network access between two VPCs.
+Databases in Flexible and Annual subscriptions are almost always deployed in a Redis VPC. In most cases, you'll need to create a [VPC peering connection]({{<relref "/rc/security/vpc-peering">}}) to access these databases. A VPC peering connection allows unrestricted network access between two VPCs.
 
 How you create these connections and the features supported vary somewhat by public cloud provider. You can read about VPC usage for [AWS](#vpcs-with-aws), [GCP](#vpcs-with-gcp), and [Azure](#vpcs-with-azure) below.
 
 ### VPCs with AWS
 
-Subscriptions that run on AWS support two VPC options. To ensure that that you can securely connect to your database, you must either [create a VPC peering connection](#creating-a-vpc-peering-connection) or [deploy your subscription in your own VPC](#deploying-in-your-own-vpc).
+Subscriptions that run on AWS support two VPC options. To ensure that that you can securely connect to your database, you need to either [create a VPC peering connection]({{<relref "/rc/security/vpc-peering#aws-vpc-peering">}}) or [deploy your subscription in your own VPC](#deploy-in-your-own-vpc).
```

Comment:
@rrelledge regarding "deploy your subscription in your own VPC" this is something that today is available only to a few veteran customers and not enabled by default. Not sure if we should mention this still in the docs.

Reply:
If you think the "deploy your subscription in your own VPC" section is no longer needed, I can go ahead and remove it.